scripts/catalog-version.js

```
@@ -0,0 +1,80 @@
+ *   - Atomic write via tmp file + renameSync
+ *
+ * Error handling:
+ *   - Missing or invalid bump type -> exit 1 with usage message
+ *   - Invalid current version in package.json -> exit 1
+ *   - Write errors -> exit 1
+ */
+
+'use strict';
+
+const { readFileSync, writeFileSync, renameSync, unlinkSync } = require('fs');
+const { join } = require('path');
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)